tot/interface.chromium.go

```diff
@@ -21,6 +21,9 @@ type Chromium interface {
 	// does not exist.
 	GetTab(tabID string) (tab Tabber, err error)
 
+	// RemoveTab removes tab reference from chrome tabs list.
```

Comment:
Instead of having a separate method for this, I would prefer executing the code inline in the `Close()` method. Being able to call `RemoveTab` outside of `Close()` opens up the possibility of removing a reference to a tab from the list of references (and loosing it) without actually closing it in the browser.

Also, adding to this interface has other affects (which is why the build failed) so if it's not necessary it's best avoided.

Reply:
Yes, you're quite right, mine was definitely an "add-on hack" and not very clean at all.

I also realize that my method in removing the tab from the list wasn't the best way; and that it's better to use the "slice trick" you've done. #lessonlearned

Thanks for doing all this!

Reply:
No, your change (modifying the interface had having a separate call) was the right one because of the way interfaces are being used to mock testing. All I ended up doing was adding the slice trick. Thanks for working on this!

Reply:
Oh ok, must have read it wrong, cheers!